recipes/datadistillr/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "datadistillr" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5f00109f3839eeffe4f705d6bccf53feb42bffbe7f2a76c3a2ec643c5d33d49d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - pandas
+    - requests
+  run:
+    - python >=3.6
```

`pandas` and `requests` should be listed under `run` instead:
```suggestion
  host:
    - python >=3.6
    - pip
  run:
    - python >=3.6
    - pandas
    - requests
```